src/components/Header/Navbar/index.js

```diff
@@ -1,59 +1,64 @@
 import React from 'react'
 import { Link } from 'react-scroll'
-// import { NavLink } from 'react-router-dom'
+import { NavLink } from 'react-router-dom'
+import navItems from './navItems.json'
 
 class Navbar extends React.Component {
   constructor(props) {
     super(props)
     this.state = {
       burgerOpen: false
-    }
-    this.toggle = this.toggle.bind(this)
+    }    
+    this.toggle = this.toggle.bind(this)    
   }
   toggle() {
```

Comment:
Man kan göra denna metod till en arrow funktion då slipper man använda bind(this) kan bara vara bra att veta :)

Tex:

toggle () => {
...
}

Reply:
oh yea? :o

Reply:
Yeah :D nytt ES6 sak. Arrow functions pekar på den komponent du gör det till eller något sånt liknande så man slipper binda. Jag trycker igenom din PR inom kort.

Reply:
Vill du fixa till det i en snabb fix commit? så kan jag kika på det sen :)

Reply:
Sure kan testa, sec :)